manifests/init.pp

```diff
@@ -73,6 +73,20 @@
 #
 # $candlepin_manage_db:: Boolean indicating whether a database should be installed, this includes db creation and user
 #
+# $manage_candlepin::   Whether to manage candlepin
+#
+# $manage_qpid::        Whether to manage qpid
+#
+# $manage_pulp::        Whether to manage pulp. Currently requires $manage_application and $manage_candlepin to be true as well.
```

Comment:
> `Currently requires $manage_application and $manage_candlepin to be true as well.``

Is that true? Why is that?

Reply:
Leftover. See, comments always go out of date since there are no tests for it.